src/pages/[platform]/build-a-backend/storage/list-files/index.mdx

```diff
@@ -0,0 +1,193 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'List files',
+  description: 'List files using Amplify Storage',
+  platforms: [
+    'angular',
+    'javascript',
+    'nextjs',
+    'react',
+    'vue'
+  ]
+};
+
+export const getStaticPaths = async () => {
+  return getCustomStaticPath(meta.platforms);
+};
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      meta
+    }
+  };
+}
+
+<BlockSwitcher>
+<Block name="List Files">
```

Comment:
Non blocking comment: From a content perspective, would this be easier to read if we just inline both of the tabs (so no blockswitcher)?  That way readers can also link to the section through the url. At the moment block switchers aren't linkable